dev/src/Command/AddComponentCommand.php

```diff
@@ -111,17 +123,32 @@ protected function execute(InputInterface $input, OutputInterface $output)
                 ->render();
         }
 
-        $productHomepage = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product homepage? ')
-        );
-
-        $productDocumentation = $this->getHelper('question')->ask(
-            $input,
-            $output,
-            new Question('What is the product documentation URL? ')
-        );
+        $version = basename(dirname($proto));
+        // yaml file contains documentation_uri and
+        // it is available at the same location as proto file
+        $libraryName = strtolower($new->componentName);
```

Comment:
We may be able to do better than this - for instance, this would result in `kms_v1.yaml` instead of `cloudkms_v1.yaml`, and `functions_v1.yaml` instead of `cloudfunctions_v1.yaml`. Maybe using the shortname would be better? This would look like `$new->shortName`

Reply:
Thank you for the suggestion, this is helpful